server/odc-server/src/main/resources/config/application.yml

```diff
@@ -75,8 +75,14 @@ management:
     web:
       exposure:
         include: "business,application"
+    enabled-by-default: ${odc.system.monitor.actuator.enabled:false}
```

Comment:
The config in `application.yml` could be directly insert into `config_system_configuration`. For example, `INSERT INTO config_system_configuration(`key`, `value`, `description`) VALUES('management.endpoints.enabled-by-default', 'false', 'xxxxxx') ON DUPLICATE KEY UPDATE `id`=`id`;`

Reply:
in order to use same key to control